yarn.lock

```diff
@@ -3157,6 +3167,16 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@internationalized/message@npm:^3.1.2":
```

Comment:
@MatissJanis can you run a `yarn dedupe` so we aren't collecting multiple patch releases of the same libs in the node modules. (will also clean up the past few months, I last had a dedupe committed in jan I think)

Reply:
Good idea! https://github.com/actualbudget/actual/pull/2493/commits/6fcde538dc4e42ce46ac3a35b8420ce383959607